index.test.js

```diff
@@ -30,339 +30,467 @@ describe('Equivalency', () => {
     });
   });
 
-  describe('equivalent (default rules)', () => {
-    it('should return false when inputs are not byte-equal', () => {
-      const instance = new Equivalency();
-      const inputs = [['a', 'b']];
-      inputs.forEach(([s1, s2]) => {
-        expect(instance.equivalent(s1, s2)).toEqual({ isEquivalent: false });
```

Comment:
Lots of this PR are wrapping these tests into an `isEquivalent` describe block in this section and the "Real world usage" section. The tests are still identical.

Another note: maybe it would be nice to remove the "equivalent" in all these describe blocks and just leave whats in the parents?